test/test_streams.py

```diff
@@ -58,7 +57,7 @@ def test04_naming_of_ostringstream(self):
         # Check if the object created is equal in all three cases
         cl0 = cppyy.gbl.std.ostringstream
         cl1 = cppyy.gbl.std.basic_ostringstream['char']
-        cl1 = cppyy.gbl.std.basic_ostringstream['char', cppyy.gbl.std.char_traits['char'] , cppyy.gbl.std.allocator['char']]
+        cl2 = cppyy.gbl.std.basic_ostringstream['char', cppyy.gbl.std.char_traits['char'] , cppyy.gbl.std.allocator['char']]
```

Comment:
Why this has changed to `cl2`?

Reply:
Otherwise it is a `NameError` in line 63-64.
There was a typo previously.

Reply:
Is that an issue upstream?